scapy/contrib/bier.py

```diff
@@ -17,7 +17,7 @@
 
 from scapy.packet import Packet, bind_layers
 from scapy.fields import BitEnumField, BitField, BitFieldLenField, ByteField, \
-    ShortField, StrLenField
+    ShortField, StrLenField, XStrLenField
```

Comment:
StrLenField must be removed (there is a flake8 error message https://travis-ci.com/secdev/scapy/jobs/268379551#L657)